signer/jws.go

```diff
@@ -227,18 +259,19 @@ func validateCriticalHeaders(protectedHdr *jwsProtectedHeader) error {
 	// validate all required critical headers are present.
```

Comment:
Nit, shouldn't this comment be in line 248? Here we throw an error if header is present but not marked critical.

Reply:
Nop, this is the correct comment. We are doing two validations
1. Validate that all headers market as critical are present in the headers map.
2. Validate that all required headers(as per spec) are marked critical.